libMBIN/Source/NMS/GameComponents/GcFreighterSyncComponentData.cs

```diff
@@ -6,5 +6,6 @@ namespace libMBIN.NMS.GameComponents
 	[NMS(Size = 0x1, GUID = 0xEBE521D537747D48, NameHash = 0x32EF46396A5EA8FE)]
     public class GcFreighterSyncComponentData : NMSTemplate
     {
+	public bool Dummy;
```

Comment:
If there is nothing you don't need a dummy bool, just leave it empty! (unless there is actually a field called `dummy`...)

Reply:
It is correct. It exist!